infoblox_client/objects.py

```diff
@@ -352,18 +352,27 @@ def __repr__(self):
                    for name in self._ea_dict)
         return "EAs:{0}".format(','.join(eas))
 
+    @staticmethod
+    def _value_to_bool(value):
```

Comment:
oslo.config has a more flexible routine that does this. Is there any reason to believe we need that? Or do we know these will always be *exactly* these strings? We do not currently have a dependency on oslo.config so avoiding adding it is best.